lib/validation-stream.js

```diff
@@ -9,6 +9,12 @@ var Duplex = require('readable-stream').Duplex;
 
 var XmlParser = require('./xml-parser');
 
+/** @arg {?string} str Input string to check.
+    @return {!boolean} True if input is undefined, null, or entirely whitespace. */
+function isBlank(str) {
```

Comment:
We shouldn't need to define this as a helper function as it's only being used once. My preference is to only break out helpers when it's used at least twice/thrice or we will need it at least twice/thrice.

Instead, we can define this boolean directly in the constructor:

```js
var isBlank = /\S/.test(str);
if (isBlank) { /* ... */ }
```